content/posts/2021-02-21-how-to-deploy-blazor-webassembly-to-aws-amplify/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How to deploy Blazor WebAssembly to AWS Amplify
+description: With ASP.NET Blazor WebAssembly (WASM) you can create .NET web applications that run completely inside of the browser sandbox. The published output of a Blazor WASM project are static files. Now that you can run .NET web applications without server-side code, you can deploy these applications to various static site hosts, such as AWS Amplify.
+authorIds:
+  - watilde
```

Comment:
You might want to put your ID here :)

Reply:
Oops. embarrassing 😅
Thank you for catching that!